linode/manager
# change: [M3-8199] - Auto-populate Image Label based on linode and disk names

## Description 📝
Auto-populate the Image Label field based on the selected Linode and Disk.

## Changes  🔄
- Auto-populate image labels using the following format: `linode_label-disk_label`.
- Change field label from "Label" to "Image Label" based on UX mockups
- Add unit tests

## Target release date 🗓️
7/8

## Preview 📷
| Before  | After   |
| ------- | ------- |
| <video src="https://github.com/linode/manager/assets/122488130/f0c67060-a497-486f-8002-266adb3a45c4" /> | <video src="https://github.com/linode/manager/assets/122488130/20f200fe-b8ce-41f0-afe9-5071488062c2" /> |

## How to test 🧪

### Prerequisites
- Go to `images/create/disk`

### Reproduction steps
- Select a Linode and Disk and observe the Image Label field is not auto-populated

### Verification steps
- Select a Linode and Disk and observe the Image Label field is auto-populated in the following format: `linode_label-disk_label`.

## As an Author I have considered 🤔

*Check all that apply*
